frontend/src/component/menu/Header/Header.tsx

```diff
@@ -20,21 +22,90 @@ import { ReactComponent as UnleashLogoWhite } from 'assets/img/logoWithWhiteText
 
 import { DrawerMenu } from './DrawerMenu/DrawerMenu';
 import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
-import { useThemeStyles } from 'themes/themeStyles';
+import { flexRow, focusable, useThemeStyles } from 'themes/themeStyles';
 import { ADMIN } from 'component/providers/AccessProvider/permissions';
 import { IPermission } from 'interfaces/user';
 import { NavigationMenu } from './NavigationMenu/NavigationMenu';
 import { getRoutes } from 'component/menu/routes';
 import { KeyboardArrowDown } from '@mui/icons-material';
 import { filterByConfig } from 'component/common/util';
 import { useAuthPermissions } from 'hooks/api/getters/useAuth/useAuthPermissions';
-import { useStyles } from './Header.styles';
-import classNames from 'classnames';
 import { useId } from 'hooks/useId';
 import { IRoute } from 'interfaces/route';
 import { ThemeMode } from 'component/common/ThemeMode/ThemeMode';
 import { useThemeMode } from 'hooks/useThemeMode';
 
+const StyledHeader = styled(AppBar)(({ theme }) => ({
+    backgroundColor: theme.palette.headerBackground,
+    padding: theme.spacing(1),
+    boxShadow: 'none',
+    position: 'relative',
+    zIndex: 300,
+}));
+
+const StyledContainer = styled(Container)(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'center',
+    maxWidth: 1280,
+    [theme.breakpoints.down('md')]: {
+        padding: '0',
+    },
+}));
+
+const StyledUserContainer = styled('div')({
+    marginLeft: 'auto',
+    display: 'flex',
+    alignItems: 'center',
+});
+
+const StyledNav = styled('nav')({
+    display: 'flex',
+    alignItems: 'center',
+    flexGrow: 1,
+});
+
+const StyledUnleashLogoWhite = styled(UnleashLogoWhite)({ width: '150px' });
+
+const StyledUnleashLogo = styled(UnleashLogo)({ width: '150px' });
+
+const StyledLinks = styled('div')(({ theme }) => ({
+    display: 'flex',
+    justifyContent: 'center',
+    marginLeft: theme.spacing(3),
+    '& a': {
+        textDecoration: 'none',
+        color: 'inherit',
+        marginRight: theme.spacing(3),
+        display: 'flex',
+        alignItems: 'center',
+    },
+}));
+
+const StyledAdvancedNavButton = styled('button')(({ theme }) => ({
+    border: 'none',
+    background: 'transparent',
+    height: '100%',
+    display: 'flex',
+    fontSize: '1rem',
+    fontFamily: theme.typography.fontFamily,
+    alignItems: 'center',
+    color: 'inherit',
+    cursor: 'pointer',
+}));
+
+const styledIconProps = (theme: Theme) => ({
+    color: theme.palette.neutral.main,
+});
+
+const StyledLink = styled(Link)(({ theme }) => ({
+    ...focusable(theme),
```

Comment:
themable function usage

Reply:
Can be simplified as `const StyledLink = styled(Link)(({ theme }) => focusable(theme));`